### PR TITLE
CMFest Footer Done

### PR DESCRIPTION
**Pull request by Matthewokiemute**

### CODE REVIEW

**Comment by 0xJamin** [comment]
@Matthewokiemute  what changes did you make with so many deletions?🤔
Screenshot 2023-05-23 at 9 09 07 AM

**Comment by 0xJamin** [comment]
This branch is supposed to come come from the main branch, why do we have changes of header here?

**Comment by Matthewokiemute** [comment]
> This branch is supposed to come come from the main branch, why do we have changes of header here?

I thought you said every task should have his own branch, then when its approved it will be deleted

**Comment by Matthewokiemute** [comment]
> @Matthewokiemute what changes did you make with so many deletions?🤔 Screenshot 2023-05-23 at 9 09 07 AM

I delete the node_modules

File: components/layout/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+    { name: "Speakers", link: "/speakers" },
+    { name: "Agenda", link: "/agenda" },
+    { name: "FAQ", link: "/faq" },
+  ];
+
+  return (
+    <footer className="container-fluid w-full mx-auto fixed bottom-0 text-white left-0 bg-primary px-10 shadow-md">
+      <div className="flex flex-col md:flex-row items-center gap-4 md:items-center md:justify-between max-w-screen-2xl py-5 pt-8 md:pt-0">
+        <div className="md:flex grid grid-cols-1 text-center gap-4 md:items-center md:justify-center text-[17px]">
+          <div className="hidden md:inline-block mr-0 md:mr-32">
```
Comment (0xJamin) [inline components/layout/Footer/Footer.tsx:19]: Font size doesn't fit design

File: components/layout/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,3 +1,69 @@
+import Link from "next/link";
+import TW from "../../../public/twitter.png";
+import IG from "../../../public/instagram.png";
```
Comment (0xJamin) [inline components/layout/Footer/Footer.tsx:3]: Always export your icons as SVG so they won't lose their quality

**Review by 0xJamin — CHANGES_REQUESTED** [review]
Please fix all pending changes

**Pull request merged**